integration-tests/debezium-common/pom.xml

```diff
@@ -28,9 +28,9 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
 
-    <artifactId>camel-quarkus-integration-test-debezium-postgres</artifactId>
-    <name>Camel Quarkus :: Integration Tests :: Debezium PostgresSQL Connector</name>
-    <description>Integration tests for Camel Quarkus Debezium PostgresSQL Connector extension</description>
+    <artifactId>camel-quarkus-integration-test-debezium-common</artifactId>
+    <name>Camel Quarkus :: Integration Tests :: Debezium Connectors</name>
+    <description>Integration tests for Camel Quarkus Debezium PostgresSQL, MySQL Connector extension</description>
```

Comment:
Better to just have `Integration tests for Camel Quarkus Debezium Extensions`, else the description will get ever larger once support for other stuff is added.